include/my_stacktrace.h

```diff
@@ -59,6 +59,24 @@ void my_set_exception_pointers(EXCEPTION_POINTERS *ep);
 void my_write_core(int sig);
 #endif
 
+/* print stack traces for all threads */
+void my_pstack();
+
+/* print stack traces for all threads before assert failures */
+#define assert_with_stack_traces(expr) \
```

Comment:
So I wonder then what is next. Do we want to merge this as-is and continue to work towards digging out all assert wrappers from different imported code bases (core my_*, InnoDB, TokuDB, ???) or do we want to hold and make this all one larger task?